### PR TITLE
Fix Key Vault - Administration Sample issue #23845

### PR DESCRIPTION
**Pull request by v-hongli1**
Fix issue #23845

@scottaddie, @joshfree, @lilyjma, @AlexGhiondea, @backwind1233 for notification.

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/ReadmeSamples.java [hunk 1]
```diff
@@ -28,9 +28,17 @@
  * Class containing code snippets that will be injected to README.md.
  */
 public class ReadmeSamples {
-    private final KeyVaultAccessControlClient keyVaultAccessControlClient = new KeyVaultAccessControlClient(null);
+    private final KeyVaultAccessControlClient keyVaultAccessControlClient =
```
Comment (conniey) [inline sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/ReadmeSamples.java:31]: If you update the code snippet samples in README, you also need to regenerate it so it matches the contents in the README.

https://github.com/Azure/azure-sdk-for-java/wiki/Building#code-snippets-in-readme

File: sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/ReadmeSamples.java [hunk 2]
```diff
@@ -28,9 +28,17 @@
...
+        new KeyVaultAccessControlClientBuilder()
+            .vaultUrl("<your-access-control-key-vault-url>")
+            .credential(new DefaultAzureCredentialBuilder().build())
+            .buildClient();;
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/ReadmeSamples.java:35]: nit;
```suggestion
            .buildClient();
```

File: sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/ReadmeSamples.java (hunk 1 above)
Comment (vcolin7) [reply sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/ReadmeSamples.java:31]: @hongli750210 This means going to the README file and updating the line numbers in every code snippet to match the ones in this file.

File: sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/ReadmeSamples.java (hunk 2 above)
Comment (hongli750210) [reply sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/ReadmeSamples.java]: Fixed in the new version.

File: sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/ReadmeSamples.java (hunk 1 above)
Comment (hongli750210) [reply sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/ReadmeSamples.java:31]: Fixed in the new version.

**Comment by vcolin7** [comment]
@hongli750210 Looks like this PR needs you to merge `main` into it to fix the pipeline issues. A fix was merged a few hours ago.

**Review by vcolin7 — APPROVED** [review]

**Comment by hongli750210** [comment]
@vcolin7 `main` has been merged into branch `issue#23845` .

**Pull request merged**